dist/openshift/cincinnati.yaml

```diff
@@ -199,6 +199,23 @@ objects:
           targetPort: ${{PE_STATUS_PORT}}
       selector:
         deploymentconfig: cincinnati
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: cincinnati
+    type: Opaque
+    data:
+      gb.address: "0.0.0.0"
+      gb.status.address: "0.0.0.0"
+      gb.registry: "quay.io"
+      gb.repository: "openshift-release-dev/ocp-release"
+      gb.log.verbosity: ${{GB_LOG_VERBOSITY}}
```

Comment:
Using `${{VAR}}` is for variables which are evaluated as ints, while `${VAR}` is for strings. Notice how double curly brackets are used for ports in this file.

Reply:
Thanks for that insight!

Reply:
I double-checked this in [the documentation on parameters](https://docs.openshift.com/container-platform/3.11/dev_guide/templates.html#writing-parameters), and wanted to add my findings here for completeness.

Double curly brackets use the json/yaml value as is, and can't be used as part of a string; it has to be used as a single value. E.g. `value: ${{PARAM}}` vs `value: "${prefix}-${suffix}". 

In conclusion, it's my understanding it's not the distinction between integers and strings per-se. Although the double curly brackets can indeed be used to pass an integer value through, one could as well pass through a string value.